README.md

```diff
@@ -8,6 +8,9 @@ Developed against DotNet 5.0.
 
 ## About this sample
 
+
+| :loudspeaker: **Notice**: The PI Adapter for Structured Data Files can send CSVs (and other files types) to EDS, OCS, and PI.  https://osisoft.github.io/PI-Adapter-Structured-Data-Files-Docs/content/index.html.  It has many more features than this sample.|
```

Comment:
Consider making this an inline link in the previous sentence? Like `The [PI Adapter for Structured Data Files](https://osisoft.github.io/PI-Adapter-Structured-Data-Files-Docs/content/index.html) can send..`

Reply:
sounds good.  i also changed my 2nd sentance.  I want to encourage people to use the adapter, but in talking with the Product team they think it is good to keep the sample out there too